### PR TITLE
Fix: Route and View created for Terms and Conditions page (#566)

### PR DESCRIPTION
**Pull request by Ram81**
@deshraj  please review!

### CODE REVIEW

File: frontend/src/js/route-config/route-config.js [hunk 1]
```diff
@@ -278,11 +278,20 @@
             templateUrl: baseUrl + "/web/error-pages/error-404.html",
             title: "Error 404",
         }
+                
+        var terms_and_conditions = {
+        	name: "terms_and_conditions",
+        	url: "/tandc",
```
Comment (aka-jain) [inline frontend/src/js/route-config/route-config.js]: @Ram81 Please change URL to "/legals".

**Comment by aka-jain** [comment]
@Ram81 : please make a seperate route for t&c. You have made it for web but it should be open such that the URL will be like "localhost:8888/legals". Hope you got my point. 
Also there is no need to add html starting anfd end tags to html, plesse import it using route.

Let me know if you need more info.

**Comment by Ram81** [comment]
@aka-jain  please review!

File: frontend/src/views/web/terms-and-conditions.html [hunk 2]
```diff
@@ -1,9 +1,3 @@
...
...etur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet. Quisque rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorem. Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc.</p>
```
Comment (deshraj) [inline frontend/src/views/web/terms-and-conditions.html]: @aka-jain are we planning to have inline css?

File: frontend/src/views/web/terms-and-conditions.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (aka-jain) [inline frontend/src/views/web/terms-and-conditions.html]: Just FYI, try to ignore inline styles as much as possible though this is a dummy data but keep this in mind :)

**Comment by aka-jain** [comment]
Looks Good @Ram81 . Merging it.

File: frontend/src/views/web/terms-and-conditions.html (hunk 2 above)
Comment (aka-jain) [inline frontend/src/views/web/terms-and-conditions.html]: Commented @deshraj, that's just the dummy text. So we can ignore this.

File: frontend/src/views/web/terms-and-conditions.html (hunk 3 above)
Comment (Ram81) [inline frontend/src/views/web/terms-and-conditions.html]: understood @aka-jain , is it fine for this page or do you want me to remove it?

**Comment by Ram81** [comment]
@aka-jain  please review & let me know changes if any!

**Comment by aka-jain** [comment]
Merging it.

**Pull request merged**